src/main/scala/com/tapad/docker/DockerComposePlugin.scala

```diff
@@ -144,12 +166,18 @@ class DockerComposePluginLocal extends AutoPlugin with ComposeFile with DockerCo
   def stopRunningInstances(state: State, args: Seq[String]): State = {
     val newState = getPersistedState(state)
 
+    val isRestart = args.contains(restartIdentifier)
+    val newArgs = args.filter(_ != restartIdentifier)
+
     //If the caller supplied arguments then attempt to stop the instances provided
-    if (args.nonEmpty) {
-      stopDockerCompose(newState, args)
+    if (newArgs.nonEmpty) {
+      stopDockerCompose(newState, newArgs)
     } else {
-      //By default if no arguments are passed stop all instances from current sbt project
       val runningInstances = getServiceRunningInstanceIds(newState)
+      if (isRestart && runningInstances.size > 1)
+        throw new IllegalArgumentException("More than one running instances from current sbt project were found. " +
+          "Please specify instance id as a parameter.")
```

Comment:
This text should read:
More than one running instance from the current sbt project was detected. Please provide an Instance Id parameter to the dockerComposeRestart command specifying which instance to stop.